packages/1ui/src/components/NetworkStats/NetworkStats.tsx

```diff
@@ -0,0 +1,57 @@
+import { cn } from 'styles'
+import { formatNumber } from 'utils'
```

Comment:
Not that it's top priority at the moment but I think that this highlights the need for us to make a `utils` package where we combine all of these + Portal into one place.

Reply:
Agreed